### PR TITLE
VR > Day View > Add Event

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/129318

Adding the day view event. **Note:** The time marker has its own ticket.

### CODE REVIEW

File: src/resources/postcss/views/_day.pcss [hunk 1]
```diff
@@ -5,4 +5,95 @@
...
+		margin-bottom: var(--spacer-1);
+
+		@media (--viewport-medium) {
+			@mixin heading-5;
+			margin-bottom: var(--spacer-2);
+		}
+	}
+
+	.tribe-events-calendar-day__event-title-link {
+		border-bottom: 2px solid transparent;
```
Comment (paulmskim) [inline src/resources/postcss/views/_day.pcss]: use `@mixin anchor-default` here

Comment (paulmskim) [reply src/resources/postcss/views/_day.pcss]: let's do this for the list event too.

**Comment by juanfra** [comment]
On hold until MattCR is back and he clarifies.

File: src/views/v2/day/event/title.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
+?>
+<h3 class="tribe-events-calendar-day__event-title">
```
[the remark this answers is not in the document]
Comment (paulmskim) [reply src/views/v2/day/event/title.php]: Same note as https://github.com/moderntribe/the-events-calendar/pull/2560#discussion_r294993146

File: src/views/v2/day/event/description.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
+?>
+<div class="tribe-events-calendar-day__event-description tribe-common-b2">
```
[the remark this answers is not in the document]
Comment (paulmskim) [reply src/views/v2/day/event/description.php]: Confirmed in call with design :+1:

File: src/resources/postcss/views/_day.pcss [hunk 4]
```diff
@@ -5,4 +5,95 @@
...
+		}
+	}
+
+	.tribe-events-calendar-day__event-datetime-featured-icon {
+		margin-left: 10px;
+	}
+
+	.tribe-events-calendar-day__event-title {
+		@mixin heading;
+		@mixin heading-6;
```
[the remark this answers is not in the document]
Comment (paulmskim) [reply src/resources/postcss/views/_day.pcss]: Same note as https://github.com/moderntribe/the-events-calendar/pull/2560#discussion_r294992675

File: src/resources/postcss/views/_day.pcss [hunk 5]
```diff
@@ -5,4 +5,86 @@
...
+	.tribe-events-calendar-day__event-title {
+		margin-bottom: var(--spacer-1);
+
+		@media (--viewport-medium) {
+			margin-bottom: var(--spacer-2);
+		}
+	}
+
+	.tribe-events-calendar-day__event-title-link {
+		@mixin anchor-default;
```
Comment (paulmskim) [inline src/resources/postcss/views/_day.pcss]: actually i forgot we had `tribe-common-anchor` class defined. let's use that on the event title link instead.

File: src/views/v2/day/event/title.php [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+ */
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
+?>
+<h3 class="tribe-events-calendar-day__event-title tribe-common-h6 tribe-common-h5--min-medium">
+	<a
+		href="<?php echo esc_url( tribe_get_event_link( $event_id ) ); ?>"
+		title="<?php the_title_attribute( $event_id ); ?>"
+		rel="bookmark"
+		class="tribe-events-calendar-day__event-title-link"
```
Comment (paulmskim) [inline src/views/v2/day/event/title.php]: add `tribe-common-anchor` here.

**Pull request merged**